runtime/makelib/targets.mk.ftl

```diff
@@ -486,9 +486,15 @@ UMA_PASM_INCLUDES:=$(addprefix -I ,$(UMA_INCLUDES))
 </#if>
 
 <#if uma.spec.type.windows>
+ifdef USE_MINGW
+UMA_M4_INCLUDES:=$(UMA_C_INCLUDES)
+else
+UMA_M4_INCLUDES:=$(subst /I,-I,$(UMA_C_INCLUDES))
```

Comment:
This needs to use `patsubst` instead to avoid altering paths like `foo/Interface` to `foo-Interface`.
```
UMA_M4_INCLUDES := $(patsubst /I%,-I%,$(UMA_C_INCLUDES))
```
Also, spacing around `:=` improves readability.

Reply:
Fixed, thank you!